packages/knip/src/plugins/next/index.ts

```diff
@@ -16,7 +16,7 @@ const productionEntryFilePatternsWithoutSrc = [
   '{instrumentation,middleware}.{js,ts}',
   'app/global-error.{js,jsx,ts,tsx}',
   'app/**/{error,layout,loading,not-found,page,template}.{js,jsx,ts,tsx}',
-  'app/**/{route,default}.{js,ts}',
+  'app/**/{route,default}.{js,jsx,ts,tsx}',
```

Comment:
This is not part of this PR, please remove.

Reply:
Will do